docs/guide/hello-world.md

```diff
@@ -146,49 +147,58 @@ import React from 'react';
  */
 import Main from 'components/main';
 
-export default React.createClass( {
-	displayName: 'HelloWorld',
-} );
+export default class HelloWorld extends React.Component {
+	
+};
 ```
 
-Cool. Let's make the React component render something for us. We'll do that by adding a `render()` method that uses the "Main" component and outputs some markup. Inside the `React.createClass` object, after "displayName: 'HelloWorld',", insert before the closing "} );":
+Cool. Let's make the React component render something for us. We'll do that by adding a `render()` method that uses the "Main" component and outputs some markup. Let's add the `render()` method inside of the `React.Component` extension like so:
 
 ```javascript
+export default class HelloWorld extends React.Component {
 	render() {
 		return (
 			<Main>
 				<h1>Hello, World!</h1>
 			</Main>
 		);
 	}
+}
 ```
 
 If you want to learn more about our approach to writing React components, check out the [Components](../components.md) page.
 
 ### 2. Hook up controller
 
-Time to hook this up with our controller function. Open `/hello-world/controller.js`. Import ReactDom and React at the top of the file:
+Time to hook this up with our controller function. Open `/hello-world/controller.js`.  
+Import ReactDom, React and your new component at the top of the file:
 
 ```javascript
-var React = require( 'react' ),
-	ReactDom = require( 'react-dom' );
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+
+/**
+ * Internal dependencies
+ */
+import HelloWorld from 'my-sites/hello-world/main';
 ```
 
 Then remove the `console.log` call and enter the following instead:
 
 ```javascript
 helloWorld() {
-	const Main = require( 'my-sites/hello-world/main' );
-
 	// Render hello world...
 	ReactDom.render(
-		React.createElement( Main ),
-		document.getElementById( 'primary' )
+		<HelloWorld />, // Your component
```

Comment:
I don't think `ReactDom.render`s call is very clear in general so, given this is a hello-world guide, I wanted to just add a little extra clarity :)